--- mpeg4ip/include/mpeg4ip.h.orig	2007-04-12 00:16:37.000000000 +0200
+++ mpeg4ip/include/mpeg4ip.h	2007-04-12 00:16:55.000000000 +0200
@@ -331,12 +331,13 @@
   AUDIO_FMT_HW_AC3,
 } audio_format_t;
 
+/*
 #ifndef HAVE_STRUCT_IOVEC
 struct iovec {
   void *iov_base;
   unsigned int iov_len;
 };
 #endif
-
+*/
 #endif /* __MPEG4IP_INCLUDED__ */
 
--- mpeg4ip/lib/mp4v2/Makefile.am.orig	2008-11-04 21:34:49.000000000 +0100
+++ mpeg4ip/lib/mp4v2/Makefile.am	2008-11-08 20:33:01.000000000 +0100
@@ -1,12 +1,12 @@
-SUBDIRS = . test util
+#SUBDIRS = . test util
 
-INCLUDES = -I$(top_srcdir)/include
+INCLUDES = -I$(top_srcdir)/mpeg4ip/include
 
-AM_CXXFLAGS = @BILLS_CPPWARNINGS@
+#AM_CXXFLAGS = @BILLS_CPPWARNINGS@
 
-lib_LTLIBRARIES = libmp4v2.la 
+noinst_LTLIBRARIES = libmp4v2.la 
 
-include_HEADERS = mp4.h 
+noinst_HEADERS = mp4.h 
 
 libmp4v2_la_SOURCES = \
 	3gp.cpp \
